src/style.css

```
@@ -0,0 +1,7 @@
+.block + .block {
```

If you want not include the first:
```suggestion
.block:not(:first-child)
```